osm2lanes/src/transform/tags_to_lanes/modes/foot_shoulder.rs

```diff
@@ -38,7 +38,9 @@ impl LaneBuilder {
 }
 
 enum Sidewalk {
-    None,
+    /// No information. This variant isn't called `None`, because that could be confused with
+    /// `sidewalk=none`.
+    Unknown,
```

Comment:
I like this change.

Reply:
I was very confused for a good few minutes